dict_test.go

```diff
@@ -111,6 +111,19 @@ func TestOmit(t *testing.T) {
 	}
 }
 
+func TestGet(t *testing.T) {
+	tests := map[string]string{
+		`{{- $d := dict "one" 1 }}{{ get $d "one"  -}}`:          "1",
+		`{{- $d := dict "one" 1 "two" "2" }}{{ get $d "two" -}}`: "2",
+		`{{- $d := dict }}{{ get $d "two" -}}`:                   "",
```

Comment:
Both of the error runs (travis and appveyor) are on Go 1.12. Something changed between Go 1.12 and Go 1.13 that appears to stop this for Go 1.13. My quick guess is that `dict` requires at least one input. It assumes the length of the inputs to be 1 or more.

Reply:
Well that was a weird one. Sorry about all the force pushes. Commit structure remained unchanged.(so no need to re-review everything.)